pkg/auth/handler/webapp/authflowv2/settings_mfa_create_oob_otp.go

```diff
@@ -39,20 +41,25 @@ func ConfigureAuthflowV2SettingsMFACreateOOBOTPRoute(route httproute.Route) http
 type SettingsMFACreateOOBOTPViewModel struct {
 	OOBAuthenticatorType model.AuthenticatorType
 	Channel              model.AuthenticatorOOBChannel
+	IsWhatsappEnabled    bool
```

Comment:
Instead of adding IsWhatsappEnabled, we should make `Channel` respect the config.

1. `whatsapp_sms` -> channel=whatsapp
2. `sms` -> channel=sms
3. `whatsapp` -> channel=whatsapp

The point is, we do not leak the logic to the template.

Reply:
Updated and added handling in phone identity as well 🙏

Reply:
Found issue with identity, fixing